conanfile.py

```diff
@@ -4,7 +4,7 @@
 class BranchioConan(ConanFile):
     scm = {
         "type": "git",
-        "url": "auto",
+        "url": "https://github.com/BranchMetrics/cpp-branch-deep-linking-attribution",
```

Comment:
This was being expanded in bintray to `git@github.com:BranchMetrics/cpp-branch-deep-linking-attribution` from my local checkout when I used the `auto` keyword here. Using the https URL seems safer.